packages/button/src/FzButton.vue

```diff
@@ -9,9 +9,11 @@
         />
       </slot>
     </div>
-    <slot>
-      {{ label }}
-    </slot>
+    <div :class="['truncate', containerClass]">
```

Comment:
For now it's all right, I would later on define a standard for all class props across all components. For example, in this case I wouldn't be able to get rid of the truncate class if I wanted to, an idea could be an override prop.

Reply:
In my opinion truncate should not be overridden because it is defined as a rule in the design system. In other cases I agree with you and we should study how to override some props/style.